class.jetpack-gutenberg.php

```diff
@@ -326,6 +326,20 @@ public static function get_jetpack_gutenberg_extensions_whitelist() {
 		return $preset_extensions;
 	}
 
+	/**
+	 * Returns a diff from a combined list of whitelisted extensions and extensions determined to be excluded
+	 *
+	 * @param  array $whitelisted_extensions An array of whitelisted extensions.
+	 *
+	 * @return array A list of blocks: eg [ 'publicize', 'markdown' ]
+	 */
+	public static function get_available_extensions( $whitelisted_extensions = null ) {
```

Comment:
Any particular reason to prefer `null` than `array()` here? If we had `array()` we could:
* Check for emptiness easier - an empty array evaluates as `false`, and a non-empty one evaluates as true.
* Be more specific about the type of data we're expecting here.
* Make sure that we're always using the same data type for this argument.

```suggestion
	public static function get_available_extensions( $whitelisted_extensions = array() ) {
```

Reply:
Yes. While making `$whitelisted_extensions` be an `array` does make some sense so that it's the same type of data, that also presents us an issue. If we were to change `$whitelisted_extensions` to default to `array`, then we have no way to tell why the array is empty. For example, is it empty because a test forced it to be empty, in which case we should populate it with data. Or, is it empty because that's the default value, in which case we should populate it with data.

Reply:
Thanks for clarifying - makes sense to me 👍